docs/docs/examples/model-inference/Huggingface-Model-Inference/index.md

```diff
@@ -59,83 +56,67 @@ if __name__ == "__main__":
 
 ## Building the container (optional)
 
-### Prerequisite
-- Install Docker on your local machine.
-- Sign up for a DockerHub account if you don't already have one.
-Steps
-
-Step 1: Create a Dockerfile
-Create a new file named Dockerfile in your project directory with the following content:
-```Dockerfile
-FROM huggingface/transformers-pytorch-deepspeed-nightly-gpu
-
-RUN apt-get update -y
-
-RUN pip -q install git+https://github.com/huggingface/transformers
-
-RUN pip -q install accelerate>=0.12.0 
-
-WORKDIR /
-
-# COPY ./dolly_inference.py .
-```
-This Dockerfile sets up a container with the necessary dependencies and installs the Segment Anything Model from its GitHub repository.
-
-Step 2: Build the Docker Image
-In your terminal, navigate to the directory containing the Dockerfile and run the following command to build the Docker image:
-
-```bash
-docker build -t your-dockerhub-username/sam:lite .
-```
-Replace your-dockerhub-username with your actual DockerHub username. This command will build the Docker image and tag it with your DockerHub username and the name "sam".
-
-Step 3: Push the Docker Image to DockerHub
-Once the build process is complete, Next, push the Docker image to DockerHub using the following command:
-```bash
-docker push your-dockerhub-username/sam:lite
-```
+You may want to create your own container for this kind of task. In that case, use the instructions for [creating](https://docs.docker.com/get-started/02_our_app/) and [publishing](https://docs.docker.com/get-started/04_sharing_app/) your own image in the docker hub. Use ` huggingface/transformers-pytorch-deepspeed-nightly-gpu` as base image, install dependencies listed above and copy the `inference.py` into it.
```

Comment:
> Use ` huggingface/transformers-pytorch-deepspeed-nightly-gpu` as base image, install dependencies listed above and copy the `inference.py` into it.

The now deleted section had a Dockerfile which does this on behalf of the user, so we have removed useful information and now left it up to them to go and finnd out how to build an appropriate dockerfile.  Should we re-include

```Dockerfile
FROM huggingface/transformers-pytorch-deepspeed-nightly-gpu
RUN apt-get update -y
RUN pip -q install git+https://github.com/huggingface/transformers
RUN pip -q install accelerate>=0.12.0 
COPY ./inference.py .
```

Reply:
Added back the Dockerfile content